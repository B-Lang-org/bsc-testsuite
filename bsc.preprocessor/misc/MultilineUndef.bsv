`define FOO

`undef

FOO

`ifndef FOO
Bool x = True;
`endif


