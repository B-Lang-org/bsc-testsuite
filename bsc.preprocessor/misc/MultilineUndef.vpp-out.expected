 `line(HERE///MultilineUndef.bsv,2,1,0)


 `line(HERE///MultilineUndef.bsv,7,12,0)
Bool x = True;
 `line(HERE///MultilineUndef.bsv,9,7,0)



